modules/migration-using-must-gather.adoc

```diff
@@ -8,20 +8,23 @@
 [id="migration-using-must-gather_{context}"]
 = Using the must-gather tool
 
-You can collect logs, metrics, and information about {local-product} custom resources by using the `must-gather` tool.
+You can collect logs and information about {local-product} custom resources by using the `must-gather` tool.
 
 The `must-gather` data must be attached to all customer cases.
 
+The `must-gather` tool is a container and does not run all the time. The tool runs for a few minutes only after you invoke the tool by running the `must-gather` command.
+
 ifdef::troubleshooting-3-4,troubleshooting-mtc[]
 You can collect data for a one-hour or a 24-hour period and view the data with the Prometheus console.
 endif::[]
 ifdef::oadp-troubleshooting[]
-You can run the `must-gather` tool with the following data collection options:
+You can run the `must-gather` tool with the following options. To use an option, you can add a flag corresponding to that option in the `must-gather` command.
+
+Default configuration:: This configuration collects pod logs, {oadp-short} and `Velero` custom resource (CR) information for all namespaces where the {oadp-short} Operator is installed.
+Timeout:: Data collection can take a long time if there are many failed `Backup` CRs. You can improve performance by setting a timeout value.
```

Comment:
Nit - per the [ISG word usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#m) entry on "might", use "might" to indicate possibility and "can" to indicate ability (as in, something "might" happen, and users "can" do something):
```suggestion
Timeout:: Data collection might take a long time if there are many failed `Backup` CRs. You can improve performance by setting a timeout value.
```